src/entities/Discourse/model.ts

```diff
@@ -0,0 +1,33 @@
+import { Model } from 'decentraland-gatsby/dist/entities/Database/model'
+import { SQL, join, table } from 'decentraland-gatsby/dist/entities/Database/utils'
+
+import { DiscourseAttributes } from './types'
+
+export default class DiscourseModel extends Model<DiscourseAttributes> {
```

Comment:
How about changing this to UserModel so we can expand it later with the other connections?